### PR TITLE
Change link from home to logout for AB#12400

### PR DESCRIPTION
**Pull request by sslaws**
# Fixes or Implements [AB#12400](https://dev.azure.com/qslvic/304a1f8c-dace-4f85-adf3-bf563d5b3a39/_workitems/edit/12400)

## Description

Adds a logout button

Behaves as per Health Gateway and takes you back to HG Landing.

image

If you are reviewing this PR, please adhere to the guidelines as [documented](https://github.com/bcgov/healthgateway/wiki/prguidance).

## Testing

-   [ ] Unit Tests Updated
-   [ ] Functional Tests Updated
-   [X] Not Required

### UI Changes

Yes

### Browsers Tested

-   [X] Chrome
-   [ ] Safari
-   [ ] Edge
-   [ ] Firefox

Provide an explanation for any test(s) not completed.

## Notes/Additional Comments

Any additional notes or comments that may be relevant. Include any specialized deployment steps here.

## Items to Review:

-   [General PR Guidelines](https://github.com/bcgov/healthgateway/wiki/PRguidance)

-   Fulfills Work Item Requirements

    -   The changes meet/implement story's acceptance criteria. Fixes identified problems if bug.
    -   Changes not directly related to the task requirements need to be documented on the PR.

-   Compilation / Tests

    -   The changes do not break compilation and/or unit or functional tests; unless other item addresses them specifically.

-   Logic Problems / Functional Behaviour

    -   The changes work as intended and do not introduce problems.

-   Performance

    -   The changes do not introduce obvious performance issues.

-   Documented Standards

    -   The changes adhere to the team's documented [Coding Standards](https://github.com/bcgov/healthgateway/wiki/standards).

-   Readability / Maintainability
    -   The changes can be easily understood/read and allow for future enhancements without major refactoring. Readability is preferred over "clever code".
    -   Reasoning for changes needs to be clearly articulated. Disagreements should be arbitrated by a third developer.

### CODE REVIEW

File: Apps/WebClient/src/ClientApp/src/views/pcrTest.vue [hunk 1]
```diff
@@ -1141,15 +1141,15 @@ export default class PcrTestView extends Vue {
...
                         <div class="pt-2">
-                            <router-link to="/">
+                            <router-link to="/logout">
                                 <hg-button
-                                    id="btn-continue"
-                                    aria-label="Continue"
-                                    data-testid="btn-continue"
+                                    id="logoutBtn"
+                                    aria-label="Logout"
+                                    data-testid="logoutBtn"
```
Comment (MikeLyttle) [inline Apps/WebClient/src/ClientApp/src/views/pcrTest.vue:1148]: Nit: `id` and `data-testid` values should be in kebab case.

File: Apps/WebClient/src/ClientApp/src/views/pcrTest.vue [hunk 2]
```diff
@@ -1141,15 +1141,15 @@ export default class PcrTestView extends Vue {
...
                                     variant="link"
                                     class="continue-button"
                                 >
-                                    <span>Back to home</span>
+                                    <span>Logout</span>
```
Comment (MikeLyttle) [inline Apps/WebClient/src/ClientApp/src/views/pcrTest.vue:1152]: Nit: The action is two words (log out).

**Review by MikeLyttle — CHANGES_REQUESTED** [review]
The link behaviour should only be changed for logged-in users; other users should continue to see the "Back to home" link.

File: Apps/WebClient/src/ClientApp/src/views/pcrTest.vue (hunk 1 above)
Comment (ninosamson) [reply Apps/WebClient/src/ClientApp/src/views/pcrTest.vue:1148]: Agree with mike on the "Log out"

**Review by ninosamson — APPROVED** [review]

File: Apps/WebClient/src/ClientApp/src/views/pcrTest.vue (hunk 1 above)
Comment (sslaws) [reply Apps/WebClient/src/ClientApp/src/views/pcrTest.vue:1148]: Keeping id and data-testid the same as other code.

**Review by MikeLyttle — APPROVED** [review]

**Pull request merged**